src/api/interfaces/frame_system.rs

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Minor naming convention: in rust you talk about traits, not interfaces.

Second, with this name, I would expect only the trait declarations, not the implementations here.

Reply:
Maybe call it `rpc_implementations`?

Reply:
I was not thinking about rust interfaces (e.g. traits) but rather the node interface. But, agreed, it is confusing.

Reply:
How about `rpc_interfaces`? It's not really an implementation, I think. Only a helper for easy rpc access and forwarding it to the client.

Reply:
Or `rpc_helpers`?

Reply:
I would still vote for `rpc_implementations` they are not helpers for the RPC, they are actual (client) implementations in my opinion. Or maybe `rpc-api`?

Reply:
👍 rpc-api